### DISCUSSION TITLE
More defined voting system

### DISCUSSION THREAD

**Discussion by zsarnett** [community/community#10] [Discussions] [answered]
Discussions on Github are great and I think they will prove to be a worthwhile investment, but I think there could be a more defined way to vote for the discussion.

Example:

    Discussion: I believe we should add this feature
    
In this example, there are probably many people in the organization that would love this feature. Currently, the only way to "vote" is to react with the Thumbs up emoji or comment "+1". And we all know those are not helpful comments to the discussion. But users do this because they aren't certain how or if the reaction of an emoji counts as voting...

Have a defined Vote button would remove the guesswork for a user on how to back this request/discussion.

A simple Vote button with a counter would suffice.
![image](https://user-images.githubusercontent.com/18730868/96269541-1d869a80-0f90-11eb-8900-7e647230eb4a.png)

**Comment by bramkragten** [comment]
I would like to vote for this.

In some previous versions of discussions there were points and bumps, but it was removed as it brought confusion.

![image](https://user-images.githubusercontent.com/5662298/96271081-bfc47380-0fcc-11eb-8acd-6aeb981ec838.png)

**Comment by becca** [comment, marked as answer]
👋 thank you both so much for this feedback. we hear you and agree that there could be another way to better signal which questions and/or comments in the Discussions space need more recognition and value added to it. Reactions are a way to emote and won't suffice.

this is definitely an area of the Discussions product we're hopeful you can see some changes soon. 🚀

**Reply by becca** [reply]
We will be enabling the Upvoting feature on this repository that @bdresser mentioned over in https://github.com/github/feedback/discussions/10#discussioncomment-183102 🚀

**Comment by scorsi** [comment]
The voting system could be activated/disactivated and if activated, displayed in the discussions lists ?

**Comment by Christophvh** [comment]
This would also allow discussions to be used as public roadmaps. Which seems a perfect fit! Emoji's is not good because people can use different ones, which makes it impossible for sorting

**Comment by bdresser** [comment]
Hi folks! We've been working on a voting feature that's now available in limited beta. If you're interested in trying it out, leave a reply below with your repository's name and we'll get you added in the next week or so. 

If you'd like to see the feature in action, head over to the [GitHub Universe discussions](https://github.com/githubevents/universe2020/discussions).

**Reply by MichaelMure** [reply]
Me, me! https://github.com/MichaelMure/git-bug

**Reply by alexfinnarn** [reply]
https://github.com/CUCentralAdvancement/giving-frontend

**Reply by czyzby** [reply]
@bdresser Is it possible to limit the voting feature to specific categories? For example, I'd like the voting feature to be available in Ideas, but not necessarily in Q&A.

**Reply by iSoron** [reply]
https://github.com/iSoron/uhabits

**Reply by eliecharra** [reply]
Can't wait to try it ! https://github.com/cloudskiff/driftctl

**Comment by phoerious** [comment]
I see that there is a voting button for posts, but it is pretty redundant with the thumbs-up reaction. Even here I see most people using thumbs-up reactions, because they have a much clearer affordance. However, it is not clear if that does anything besides putting an emoji on the post (probably not), so do I have to use the vote button as well? What happens if I vote up, but give thumbs down? Why do I have to click two buttons to do the same thing? Do people even realise that there is an additional vote button? It is very muted and totally not obvious to see.

**Reply by zsarnett** [reply]
Voting for something and reacting to a post is different to a degree. And I think a good amount of the reactions were made before the voting system

**Reply by phoerious** [reply]
The voting button is way too small and easy to overlook and I don't really see a difference between thumbs up/vote up and thumbs down/vote down. Yes, there are more emoji reactions than those two, but thumbs up and down is IMHO entirely equivalent to voting.

**Reply by zsarnett** [reply]
IMO this is a new solution. It will take time to use yes. But its just different. 

Should Emojies be removed. Maybe. Maybe an option to do that. But it will just be a new way that people will have to get used to

**Reply by phoerious** [reply]
I just don't like that there are two ways to do the same thing but with different effects. You should be using the voting function to increase the visibility of good posts or solutions, but that won't happen if people use the thumbs up instead. The intention is the same, but the result is different and it is not obvious to the user why. And clicking both is simply redundant. So either one of those should be removed or a thumbs up/down should automatically also trigger a vote.
